src/test/java/org/sagebionetworks/web/unitclient/presenter/ACTDataAccessSubmissionsPresenterTest.java

```diff
@@ -94,25 +102,54 @@
 	GWTWrapper mockGWT;
 	@Mock
 	DateTimeFormat mockDateTimeFormat;
+	@Mock
+	SynapseSuggestBox mockAccessorSuggestWidget;
+	@Mock
+	UserGroupSuggestionProvider mockUserGroupSuggestionProvider;
+	@Mock
+	UserBadge mockSelectedAccessorUserBadge;
+	@Mock
+	UserGroupSuggestion mockUserGroupSuggestion;
+	@Mock
+	UserGroupHeader mockUserGroupHeader;
+
+	
 	@Captor
 	ArgumentCaptor<Place> placeCaptor;
 	public static final String FILE_HANDLE_ID = "9999";
 	public static final String AR_ID = "76555";
+	public static final String ACCESSOR_ID = "9999999";
 	public static final String NEXT_PAGE_TOKEN = "abc678";
+	public static final Long EXPIRATION_PERIOD = 0L;
+	public static final String USER_ID_SELECTED = "42";
 
+	
 	@Before
 	public void setup() {
 		MockitoAnnotations.initMocks(this);
 		when(mockGWT.getDateTimeFormat(any(PredefinedFormat.class))).thenReturn(mockDateTimeFormat);
-		presenter = new ACTDataAccessSubmissionsPresenter(mockView, mockSynAlert, mockGinInjector, mockLoadMoreContainer, mockACTAccessRequirementWidget, mockButton, mockDucTemplateFileHandleWidget, mockJsClient, mockSubjectsWidget, mockGWT);
+		presenter = new ACTDataAccessSubmissionsPresenter(mockView, mockSynAlert, mockGinInjector, mockLoadMoreContainer, mockACTAccessRequirementWidget, mockButton, mockDucTemplateFileHandleWidget, mockJsClient, mockSubjectsWidget, mockGWT, mockAccessorSuggestWidget, mockUserGroupSuggestionProvider, mockSelectedAccessorUserBadge);
 		AsyncMockStubber.callSuccessWith(mockACTAccessRequirement).when(mockJsClient).getAccessRequirement(anyString(), any(AsyncCallback.class));
-		AsyncMockStubber.callSuccessWith(mockDataAccessSubmissionPage).when(mockJsClient).getDataAccessSubmissions(anyString(), anyString(), any(SubmissionState.class), any(SubmissionOrder.class), anyBoolean(), any(AsyncCallback.class));
+		AsyncMockStubber.callSuccessWith(mockDataAccessSubmissionPage).when(mockJsClient).getDataAccessSubmissions(anyString(), anyString(), anyString(), any(SubmissionState.class), any(SubmissionOrder.class), anyBoolean(), any(AsyncCallback.class));
 		when(mockDataAccessSubmissionPage.getResults()).thenReturn(Collections.singletonList(mockDataAccessSubmission));
 		when(mockDataAccessSubmissionPage.getNextPageToken()).thenReturn(NEXT_PAGE_TOKEN);
 		when(mockACTAccessRequirement.getDucTemplateFileHandleId()).thenReturn(FILE_HANDLE_ID);
 		when(mockACTAccessRequirement.getId()).thenReturn(Long.parseLong(AR_ID));
 		when(mockGinInjector.getACTDataAccessSubmissionWidget()).thenReturn(mockACTDataAccessSubmissionWidget);
 		when(mockACTAccessRequirement.getSubjectIds()).thenReturn(mockSubjects);
+		
+		when(mockACTAccessRequirement.getDucTemplateFileHandleId()).thenReturn(FILE_HANDLE_ID);
```

Comment:
Moved mockACTAccessRequirement setup into here since multiple tests now rely on it.